### ISSUE TITLE
missing migration for iconfont.identifier and icon font.zip_file

### ISSUE THREAD

**Issue by electroniceagle** [jrief/djangocms-cascade#218]
We're getting a missing migration for identifier and zip_file fields of the iconfont model.

$ pip freeze | grep cascade
djangocms-cascade==0.11.2

```
$ more ~/.virtualenvs/shop-core/lib/python2.7/site-packages/cmsplugin_cascade/migrations/0015_auto_20170111_1300.py
# -*- coding: utf-8 -*-
# Generated by Django 1.9.12 on 2017-01-11 19:00
from __future__ import unicode_literals

from django.db import migrations, models
import django.db.models.deletion
import filer.fields.file


class Migration(migrations.Migration):

    dependencies = [
        ('cmsplugin_cascade', '0014_glossary_field'),
    ]

    operations = [
        migrations.AlterField(
            model_name='iconfont',
            name='identifier',
            field=models.CharField(help_text='A unique identifier to distinguish this icon font.', max_length=50, unique=True, verbose_name='Identifier'),
        ),
        migrations.AlterField(
            model_name='iconfont',
            name='zip_file',
            field=filer.fields.file.FilerFileField(help_text='Upload a zip file created on <a href="http://fontello.com/" target="_blank">Fontello</a> containing fonts.', on_delete=django.db.models.deletion.CASCADE, to='filer.File'),
        ),
    ]
```

**Comment by jrief** [comment]
djangocms-cascade version 0.11.2 is not shipped with a file named ``migrations/0015_auto_20170111_1300.py``. Moreover that file was ``Generated by Django 1.9.12 on 2017-01-11 19:00``. May it be that you generated that file yourself?

**Comment by rfleschenberg** [comment]
The bug is that we don't ship the migration, so running `manage.py makemigrations` will generate it. It might be good to check for missing migrations on travis.

**Comment by rfleschenberg** [comment]
I can confirm the bug. Should be an easy fix. @electroniceagle, thanks for the report!

**Comment by electroniceagle** [comment]
Sorry, had to run out to pick up my daughter at school and pushed this out without any explanation.  Yes, the fields were added here https://github.com/jrief/djangocms-cascade/commit/3352a38f57497b5215cceb8da57ed0c164ddc0c8 without a migration.

Here is a test we do in our sites looking for missing migrations.
```
# This checks if migrations are missing
# --exit is backwards, it returns 1 if no migrations needed
.PHONY: testmigrations
testmigrations: check-venv clean
        ./manage.py makemigrations --dry-run --noinput -v 3 --exit ; test 1 -eq $$?
```

**Comment by rfleschenberg** [comment]
@electroniceagle If you have the time to, it would be totally awesome if you file a PR that updates our CI to check for this. See #219. If you don't get around to do it, no problem, I will do it sooner or later then. Cheers!

**Comment by rfleschenberg** [comment]
@jrief I suggest we release 0.11.3 with this fix before it bites more users. What do you think?

**Comment by jrief** [comment]
Just released 0.11.3 with a fix for this.

**Comment by rfleschenberg** [comment]
@jrief Is modifying the old migration the right fix? Would it be better to merge #220 instead and ship the new migration? I'm not sure right now.
